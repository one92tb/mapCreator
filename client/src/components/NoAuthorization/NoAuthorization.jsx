import React from "react";
import { Wrapper, TextBox } from "./style";

class NoAuthorization extends React.Component {
  render() {
    return (
      <Wrapper>
        <TextBox>
          You have no permission to access this components. You must have admin
          status.
        </TextBox>
      </Wrapper>
    );
  }
}

export default NoAuthorization;
