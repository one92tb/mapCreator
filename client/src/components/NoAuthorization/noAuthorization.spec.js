import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import { NoAuthorization } from "./NoAuthorization";

describe("no authorization component", () => {
  it("shallow authorization component", () => {
    let wrapper;

    wrapper = shallow(<NoAuthorization />);
    expect(wrapper).toMatchSnapshot();
  });
});
