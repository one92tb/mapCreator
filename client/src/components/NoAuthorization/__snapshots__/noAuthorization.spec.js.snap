// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`no authorization component shallow authorization component 1`] = `
<div>
  <div>
    You have no permission to access this components. You must have admin status.
  </div>
</div>
`;
